packages/twenty-emails/tsup.index.tsx

```diff
@@ -1,2 +1,4 @@
 export * from './src/emails/clean-inactive-workspaces.email';
 export * from './src/emails/delete-inactive-workspaces.email';
+export * from './src/emails/password-reset-link.email';
+export * from './src/emails/password-update-notify.email';
```

Comment:
new line missing

Reply:
Not able to put new line, it was being removed on saving the file in VSCode. Can we skip this?

Reply:
yes np